### ISSUE TITLE
Enderal Forggoten Stories Support / Nemesis Crash on Update

### ISSUE THREAD

**Issue by Sarund9** [ShikyoKira/Project-New-Reign---Nemesis-Main#132]
More of a Feature request, current behabior engine does not support the Steam version of Enderal.
The Steam version installs enderal to a Separate folder, Nemesis detected this as a Wrong installation the first time i Launched.
Now Nemesis crashes when it Updates, likely because of the same reason.

**Comment by ShikyoKira** [comment]
whats the file structure for steam version of Enderal? Where do you place your mod? What is directory path of Enderal?

Just post a few pictures of it will do

**Comment by Sarund9** [comment]
Steam installs enderal as a Separate game, in SteamApps > Common, there are your normal game folders, (Skyrim, Skyrim Special Edition, Fallout 4, Etc..). There is a Folder for Enderal.
The Folder contains an Exact Replica of Skyrim LE Installation with:
- No DLCs
- SKSE, Crash Fix Plugin, SkyUI and Other Mods are Pre-Installed
- There are no aditional Plugin files other than Skyrim.esm and Update.esm. Enderal is acctually skyrim.esm modified, and not it's own plugin file.
- ForggotenStories.esm is the DLC that comes with the Steam Version
- There are also a Custom Launcher installed

FNIS is a Part of enderal, it was used to Generate new Animations (Apparently all are Idles played for Scripted Events/Cutscenes), however, FNIS itself is not installed, and Must be installed in order to install FNIS dependant Mods. GenerateFNISforUsers tool is not included, but was used to create the behaviors for the new animations.

Note: I have installed some ENB files and Wrye Bash
![Screen Shot](https://user-images.githubusercontent.com/53350334/72669259-33b9f700-3a0e-11ea-9b4e-b040de02e979.png)

**Comment by Sarund9** [comment]
Oh sorry i didn't understand correcly, i use MO2 latest version that recognizes enderal.
Enderal is installed on HardDrive F: and the Mods directory is on E:, is this a Problem?

**Comment by ShikyoKira** [comment]
Nope. I don't think that is the problem. The file structure looks fine to me. You said it crash on update. By right, if you install it in a wrong location, it would detect that and prevent you from even updating the engine

Send me the updatelog.txt. I'll see if I can find any useful information from it

**Comment by Sarund9** [comment]
Update Log:

[2020-Jan-20 17:21:03] Nemesis Behavior Version: v0.83b
[2020-Jan-20 17:21:03] Current Directory: F:\SteamLibrary B\steamapps\common\Enderal\data\Nemesis_Engine
[2020-Jan-20 17:21:03] Data Directory: F:\SteamLibrary B\steamapps\common\Skyrim\Data\
[2020-Jan-20 17:21:03] Skyrim Special Edition: FALSE
[2020-Jan-20 17:21:03] Detecting processes...
[2020-Jan-20 17:21:03] Process count: 32
[2020-Jan-20 17:21:03] Initializing file check...
[2020-Jan-20 17:21:03] File Check complete
[2020-Jan-20 17:21:03] External script run complete
[2020-Jan-20 17:21:03] Temp behavior clearance complete
[2020-Jan-20 17:21:03] Data record complete
[2020-Jan-20 17:21:03] New Animations extraction start (Folder: behavior templates\b\animationdatasinglefile)
[2020-Jan-20 17:21:03] New Animations extraction complete (Folder: behavior templates\b\animationdatasinglefile)
[2020-Jan-20 17:21:03] New Animations extraction start (Folder: behavior templates\b\behavior)

**Comment by ShikyoKira** [comment]
It is not a bug. That message is correct. It is expecting it to be in skyrim. However, Enderal has it in different folder name so messing up Nemesis

`[2020-Jan-20 17:21:03] Current Directory: F:\SteamLibrary B\steamapps\common\Enderal\data
\Nemesis_Engine`

`[2020-Jan-20 17:21:03] Data Directory: F:\SteamLibrary B\steamapps\common\Skyrim\Data`

If you look at these 2 logs, it is trying to get skyrim's directory hence it fails to update

Anyway, I'll add Enderal support in the next update

**Comment by NielsGx** [comment]
Still doesn't work ?

Thanks for all

**Comment by M0J0MAN** [comment]
Any workaround we could possibly do on our own in the meantime? I don't really know anything but this seems just like a small blip that can be circumvented.

Also thanks so much for the work on Nemesis, recently came back to Skyrim and my playthrough on SE was a blast, always tried to avoid FNIS the previous times I set up a modlist and Nemesis was super smooth.

**Comment by ShikyoKira** [comment]
go to nemesis.ini and edit the "skyrimdatadirectory" to enderal's data directory. This will force it to redirect to enderal's data directory instead of getting skyrim's data directory

**Comment by kostyaplay2706-stack** [comment]
> It is not a bug. That message is correct. It is expecting it to be in skyrim. However, Enderal has it in different folder name so messing up Nemesis
> 
> `[2020-Jan-20 17:21:03] Current Directory: F:\SteamLibrary B\steamapps\common\Enderal\data \Nemesis_Engine`
> 
> `[2020-Jan-20 17:21:03] Data Directory: F:\SteamLibrary B\steamapps\common\Skyrim\Data`
> 
> If you look at these 2 logs, it is trying to get skyrim's data directory hence it fails to update
> 
> Anyway, I'll add Enderal support in the next update. Thanks for reporting =)
